scico/_version.py

```diff
@@ -43,14 +41,7 @@ def variable_assign_value(path: str, var: str) -> Any:
         try:
             # See http://stackoverflow.com/questions/2058802
```

Comment:
Could link directly to the answer used, which I think is https://stackoverflow.com/a/30471662